docs/src/main/asciidoc/mongodb-panache.adoc

```diff
@@ -653,17 +653,27 @@ See this part of the documentation: link:mongodb#simplifying-mongodb-client-usag
 Another option is to use the `@BsonCreator` annotation to tell the `PojoCodec` to use the Kotlin data class default constructor,
 in this case all constructor parameters have to be annotated with `@BsonProperty`: see link:{mongodb-doc-root-url}/bson/pojos/#supporting-pojos-without-no-args-constructors[Supporting pojos without no args constructor].
 
+This will only works when extending `PanacheMongoEntityBase`, and not `PanacheMongoEntity`, as the ID field also needs to be included in the constructor.
```

Comment:
```suggestion
This will only work when the entity extends `PanacheMongoEntityBase` and not `PanacheMongoEntity`, as the ID field also needs to be included in the constructor.
```